package.json

```diff
@@ -42,8 +42,9 @@
     "pkginfo": "^0.4.1",
     "vscode-debugprotocol": "^1.11.0",
     "vscode-extension-telemetry": "0.0.10",
-    "vscode-languageclient": "^2.5.0",
+    "vscode-languageclient": "^3.5.0",
```

Comment:
Why do you need the language client for the server? It should be used only by clients.

Reply:
Ah you are right, it was only used previously for the protocol structures.  Now that they are in their own package, we can remove it.  I updated the patch.